addons/account/models/account_account.py

```diff
@@ -480,6 +486,29 @@ def _compute_account_type(self):
             closest_index = bisect_left(codes_list, account.code) - 1
             account.account_type = accounts_with_codes[account.company_id.id][codes_list[closest_index]] if closest_index != -1 else 'asset_current'
 
+    @api.depends('code')
+    def _compute_account_tags(self):
```

Comment:
There is a lot of duplicated code between this compute and the _compute_account_type. I discussed with some people of the office about merge the two, we decided to split them so that the functionality are isolated from one another. But i could make a function to avoid duplicating the code.